tests/phpunit/tests/block-supports/border.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * @group block-supports
+ */
```

Comment:
It would be nice to add `@covers` tag to improve code coverage reports.
```suggestion
 * @covers ::wp_apply_border_support
 */
```